build.gradle

```diff
@@ -21,60 +20,29 @@ minecraft {
 
 dependencies {
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
-	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
-	modCompile "net.fabricmc:fabric-loader:${project.loader_version}"
+	mappings "net.fabricmc:yarn:${project.yarn_mappings}"
+	modImplementation "net.fabricmc:fabric-loader:${project.loader_version}"
 
 	
-	shadow "blue.endless:jankson:1.2.0"
-}
-
-task jar(type: Jar, overwrite: true) {
-	classifier = "compile"
-	from compileJava
-	from processResources
-}
-
-shadowJar {
-	classifier = "shadow"
-	configurations = [project.configurations.shadow]
-}
-
-task remapJar(type: net.fabricmc.loom.task.RemapJarTask, overwrite: true) {
-	dependsOn shadowJar
-	from zipTree(shadowJar.archiveFile)
-
-	classifier = "release"
-	//	input = shadowJar.archiveFile
-	//	classifier = "release"
-	//output = file("build/libs/${project.buildDir.absolutePath}/libs/${archivesBaseName}-${project.version}.jar")
+	api "blue.endless:jankson:1.2.1"
+	include "blue.endless:jankson:1.2.1"
```

Comment:
This is a really serious change, and should be thought about more seriously.

What this results in, most likely, is a mod named Jankson-Fabric-3.0.1+j1.2.0 (which is already incorrect), plus a mod named jankson-1.2.1 (which is correct).

At that rate, the only added value of *this* mod (Jankson-Fabric) is to add default serializers. That would be a project redesign. I'm not saying no, but we should talk about this.